tests/std/include/range_algorithm_support.hpp

```diff
@@ -0,0 +1,532 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+
+#include <concepts>
+#include <cstddef>
+#include <functional>
+#include <ranges>
+#include <span>
+#include <type_traits>
+#include <utility>
+
+#define STATIC_ASSERT(...) static_assert(__VA_ARGS__, #__VA_ARGS__)

```

Comment:
Do we need this macro given that we're 20 only?

Reply:
"Need" no, but it is convenient to have a message in addition to a line number when `static_assert`s fail while developing a new test. I'll yank it if it bothers you.